Makefile

```diff
@@ -225,6 +225,12 @@ lotus-stats:
 .PHONY: lotus-stats
 BINS+=lotus-stats
 
+eudico-stats:
+	rm -f eudico-stats
```

Comment:
What if someone wants to keep their metrics, will re-building `eudic-stats` remove everything? Maybe consider including this in `make clean`?

Reply:
this should just remove the command. I think in the future, the data should be persisted in the db, so building or removing should not affect. And once the db is in place, I will add back the `reset` flag so that ppl can clean the existing data.